### ISSUE TITLE
How do you train the embedding spaces(ACAI,BiGAN,DeepCluster,etc)?

### ISSUE THREAD

**Issue by lampardwk** [hsukyle/cactus-maml#14]
Thanks for your paper and code. It is really a very good work!
I have some issues regarding to embedding.The data set given in the code seems to be the result that has been encoded by embedding. How to ensure that the tasks generated by embedding are more credible? In other words, how do you train the embedding spaces(ACAI,BiGAN,DeepCluster,etc)?
Looking forward to your reply, Thank you！

**Comment by kylehkhsu** [comment]
I trained the models that produced the embedding spaces by using the respective authors' code. I held out ~10% of the training split for each dataset to serve as a validation set for tuning hyperparameters for this phase. There was no feedback from the meta-learning portion of the pipeline into the embedding function training phase (we tried a simple version of this and it didn't work).
